### DISCUSSION TITLE
[OpenAI] Server Unavailable Error, Error code: 503 - {'error': {'message': 'No available channel for model text-embedding-ada-002 under current group default (request id: 2024062108461012851345956869645)', 'type': 'v_api_error'}}

### DISCUSSION THREAD

**Discussion by jack-wang-sen** [langgenius/dify#5456] [General]
[OpenAI] Server Unavailable Error, Error code: 503 - {'error': {'message': 'No available channel for model text-embedding-ada-002 under current group default (request id: 2024062108461012851345956869645)', 'type': 'v_api_error'}}

**Comment by crazywoola** [comment]
You need to check with your provider.
